app/views/jobseekers/profiles/personal_details/_summary.html.slim

```diff
@@ -11,6 +11,7 @@ ruby:
 
   summary = govuk_summary_list(html_attributes: { id: "personal_details" }) do |summary_list|
     attributes.each do |attribute, options|
+      next if attribute == :last_name
```

Comment:
Non-blocking suggestion: 
Instead of having this conditional logic. Could the first row for the name be built separately as the `email` row is done at the end?

From a shallow look, it seems we could remove `first_name` and `last_name` from the `attributes` hash and have something like:
```ruby
summary_list.with_row do |row|
  row.with_key text: "Name"
  row.with_value text: "#{form.first_name} #{form.last_name}"
  row.with_action(text: t("buttons.change"), href: edit_personal_details_jobseekers_profile_path(step: "hardcoded_step_name"), visually_hidden_text: " name")
  
attributes.each do |attribute, options|
# Code without conditionals or skipping
...
```

For your consideration, but not blocking!